<ui version="4.0" >
 <class>mainform</class>
 <widget class="QDialog" name="mainform" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>537</width>
    <height>156</height>
   </rect>
  </property>
  <property name="sizePolicy" >
   <sizepolicy vsizetype="Preferred" hsizetype="Preferred" >
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle" >
   <string>WiiTCPLoad GUI 0.01</string>
  </property>
  <layout class="QGridLayout" >
   <item rowspan="2" row="0" column="0" >
    <layout class="QGridLayout" >
     <item row="0" column="0" >
      <widget class="QLabel" name="label" >
       <property name="text" >
        <string/>
       </property>
       <property name="pixmap" >
        <pixmap resource="../resource.qrc" >:/new/prefix1/krfb.png</pixmap>
       </property>
      </widget>
     </item>
     <item row="1" column="0" >
      <spacer>
       <property name="orientation" >
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="0" column="1" >
    <layout class="QGridLayout" >
     <item row="0" column="0" >
      <widget class="QLabel" name="label_2" >
       <property name="text" >
        <string>Wii Host</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1" colspan="2" >
      <widget class="QLineEdit" name="wiiHostName" />
     </item>
     <item row="1" column="0" >
      <widget class="QLabel" name="label_3" >
       <property name="text" >
        <string>Local file</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1" >
      <widget class="QLineEdit" name="localFile" />
     </item>
     <item row="1" column="2" >
      <widget class="QToolButton" name="openFile" >
       <property name="text" >
        <string>...</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0" >
      <widget class="QLabel" name="label_4" >
       <property name="text" >
        <string>Channel</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1" colspan="2" >
      <widget class="QComboBox" name="channelSelect" >
       <item>
        <property name="text" >
         <string>Homebrew Channel</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>TCPLoader Channel</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="3" column="1" >
      <spacer>
       <property name="orientation" >
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="1" column="1" >
    <layout class="QGridLayout" >
     <item row="0" column="0" colspan="2" >
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="1" >
      <widget class="QPushButton" name="readyBtn" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text" >
        <string>connect</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../resource.qrc" />
 </resources>
 <connections/>
</ui>
